### PR TITLE
Remove redundant code

### LINKED ISSUE #3144: Remove redundant codes
**Problem**
There are some repetitive import [here](https://github.com/Leo6Leo/eventing-kafka-broker/blob/1d806b7db0a4a3819ccb8e879b50a7fcf6b42de2/control-plane/pkg/reconciler/sink/kafka_sink_test.go#L56-L57) and [here](https://github.com/Leo6Leo/eventing-kafka-broker/blob/1d806b7db0a4a3819ccb8e879b50a7fcf6b42de2/control-plane/pkg/reconciler/sink/kafka_sink_test.go#L191-L198). And they use `reconciler.ControllerTopicOwner` and `sink.ControllerTopicOwner`.

**[Persona:](https://github.com/knative/eventing/blob/master/docs/personas.md)** 
Which persona is this feature for?

**Exit Criteria**
A measurable (binary) test that would indicate that the problem has been resolved.

**Time Estimate (optional):** 
How many developer-days do you think this may take to resolve?
1

**Additional context (optional)**
Add any other context about the feature request here.

### PR DESCRIPTION
**Pull request by Leo6Leo**
Fixes #3144 

## Proposed Changes
- Remove the redundant import
- Update the variable name to follow lint rules

### CODE REVIEW

File: control-plane/pkg/reconciler/sink/kafka_sink_test.go [hunk 1]
```diff
@@ -96,8 +95,9 @@ var (
 		Path:   fmt.Sprintf("/%s/%s", SinkNamespace, SinkName),
 	}
 
-	createTopicError = fmt.Errorf("failed to create topic")
-	deleteTopicError = fmt.Errorf("failed to delete topic")
+	errCreateTopic = fmt.Errorf("failed to create topic")
+
+	errorDeleteTopic = fmt.Errorf("failed to delete topic")
```
Comment (Cali0707) [inline control-plane/pkg/reconciler/sink/kafka_sink_test.go:100]: These two variables should probably be named consistently
```suggestion
	errorCreateTopic = fmt.Errorf("failed to create topic")

	errorDeleteTopic = fmt.Errorf("failed to delete topic")
```

File: control-plane/pkg/reconciler/sink/kafka_sink_test.go [hunk 2]
```diff
@@ -551,7 +551,7 @@ func sinkReconciliation(t *testing.T, format string, env config.Env) {
 					corev1.EventTypeWarning,
 					"InternalError",
 					"failed to create topic: %s: %v",
-					SinkTopic(), createTopicError,
+					SinkTopic(), errCreateTopic,
```
Comment (Cali0707) [inline control-plane/pkg/reconciler/sink/kafka_sink_test.go:554]:
```suggestion
					SinkTopic(), errorCreateTopic,
```

File: control-plane/pkg/reconciler/sink/kafka_sink_test.go [hunk 3]
```diff
@@ -569,14 +569,14 @@ func sinkReconciliation(t *testing.T, format string, env config.Env) {
 				},
 			},
 			OtherTestData: map[string]interface{}{
-				wantErrorOnCreateTopic: createTopicError,
+				wantErrorOnCreateTopic: errCreateTopic,
```
Comment (Cali0707) [inline control-plane/pkg/reconciler/sink/kafka_sink_test.go:572]:
```suggestion
				wantErrorOnCreateTopic: errorCreateTopic,
```

File: control-plane/pkg/reconciler/sink/kafka_sink_test.go (hunk 1 above)
Comment (Cali0707) [reply control-plane/pkg/reconciler/sink/kafka_sink_test.go:100]: Oh okay, @Leo6Leo should we rename `errorDeleteTopic` to be `errDeleteTopic` as well then?

**Review by pierDipi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**